example/scalalib/native/3-multi-module/build.mill

```diff
@@ -0,0 +1,77 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._, scalanativelib._
+
+trait MyModule extends ScalaNativeModule {
+  def scalaVersion = "2.13.11"
+  def scalaNativeVersion = "0.5.5"
+  
+  object test extends ScalaNativeTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+}
+
+object foo extends MyModule {
+  def moduleDeps = Seq(bar)
+  def ivyDeps = Agg(ivy"com.lihaoyi::mainargs::0.7.6")
+
+  os.proc("mkdir", millSourcePath.toString + "/target").call(stdout = os.Inherit)
+
+  os.proc(
+    "gcc", "-m64", "-shared", "-fPIC", 
+    millSourcePath.toString + "/native-src/HelloWorldFoo.c", millSourcePath.toString + "/native-src/htm.c",
+    "-o", millSourcePath.toString + "/target/libHelloWorldFoo.so")
+  .call(stdout = os.Inherit)
+}
```

Comment:
This should be in a `Task`, and probably moved to the `trait MyModule` so you don't need to copy-paste it